test/app/fiber.result

```diff
@@ -13,6 +13,22 @@ env = require('test_run')
 test_run = env.new()
 ---
 ...
+--
+-- gh-4265: Printing a finished fiber should not raise an error. This testcase
+-- is placed at the beginning to avoid updating f.id on any change in the test.
+--
+f = fiber.create(function() end)
+---
+...
+f
+---
+- status: dead
+  id: 117
+...
+tostring(f)
+---
+- fiber 117 is dead
+...
```

Comment:
Extension of diff tests is discouraged. Let's add a new test file `app-luatest/fiber_test.lua` and add a couple of tests cases to it `test_serialize` and `test_tostring`, where we'd test these functions extensively (both dead and alive fibers).

Reply:
Done.